components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -312,6 +316,21 @@ abstract class AbstractFetchDownloadService : Service() {
         }
     }
 
+    /**
+     * If notification accent color provided from browser side, returns it. Otherwise returns
+     * default accent color.
+     */
+    data class Style(
+        @ColorRes
+        val notificationAccentColor: Int? = null
+    ) {
+        internal fun getNotificationAccentColor(context: Context): Int {
+            return ContextCompat.getColor(
+                context,
+                notificationAccentColor ?: R.color.mozac_feature_downloads_notification)
+        }
```

Comment:
if we make `notificationAccentColor` not null this will be not need for this function :)